internal/backends/linuxkms/calloop_backend/input.rs

```diff
@@ -58,7 +58,7 @@ pub struct LibInputHandler<'a> {
     mouse_pos: Pin<Rc<Property<Option<LogicalPosition>>>>,
     last_touch_pos: LogicalPosition,
     window: &'a i_slint_core::api::Window,
-    keystate: xkb::State,
+    keystate: std::cell::OnceCell<xkb::State>,
```

Comment:
nitpick: Since keystate is only initialized in a location where this sturct is mut, it could be a `Option` instead of a OnceCell.
and then you can do `Option::get_or_insert_with`  to avoid the unwrap() after get_mut.

Reply:
That’s an excellent idea - thanks! Will change.